.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -3,4 +3,13 @@ blank_issues_enabled: false
 contact_links:
 - name: Discourse
   url: https://discourse.brew.sh
-  about: Not sure if it's a bug that affects everyone? The quickest way to get help is on Discourse!
+  about: Have a question? Not sure if your issue affects everyone reproducibly? The quickest way to get help is on Homebrew's Discourse!
+- name: Homebrew/homebrew-core
+  url: https://github.com/Homebrew/homebrew-core/issues/new/choose
+  about: On macOS/Mac OS X? Having a `brew` problem with a `brew install` or `brew upgrade` of a single formula/package? Report it to Homebrew/homebrew-core (the macOS core tap/repository).
```

Comment:
~~Mac OS X~~

Reply:
Specifically keeping this here for people on old versions and/or confused about names.